build.gradle

```diff
@@ -63,6 +63,7 @@ task systemTest(type: Test, description: 'Runs the system tests.', group: 'Verif
     systemProperties System.properties.findAll { property ->
         property.key.startsWith("systemTest.")
     }
+    systemProperty 'kubernetes.websocket.ping.interval', 30000
```

Comment:
Is this something you could stick in the properties config file? The only reason the other ones are here are for override purposes...

Reply:
I tried that, the properties files don't load the values into System.properties where kubernetes looks for it

Reply:
Ah, bummer.